mock-core-configs/etc/mock/templates/fedora-eln.tpl

```diff
@@ -12,17 +12,24 @@ config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils f
 
 config_opts['dist'] = 'eln'  # only useful for --resultdir variable subst
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
-config_opts['package_manager'] = 'dnf'
+
+# https://fedoraproject.org/wiki/Changes/BuildWithDNF5
+# https://pagure.io/releng/issue/11895
+config_opts['package_manager'] = 'dnf5'
 
 # Per https://github.com/fedora-eln/eln/issues/164 updated up to 4 times a day.
 # Docs: https://docs.fedoraproject.org/en-US/eln/deliverables/#_container_image
 config_opts['bootstrap_image'] = 'quay.io/fedoraci/fedora:eln'
-# Per https://github.com/fedora-eln/eln/issues/166
-config_opts['bootstrap_image_ready'] = True
+
+# Used to be True per https://github.com/fedora-eln/eln/issues/166 but with
+# the 'dnf5' we are switching to False, reobserve after
+# https://fedoraproject.org/wiki/Changes/ReplaceDnfWithDnf5 applied to ELN!
+config_opts['bootstrap_image_ready'] = False
```

Comment:
Unfortunately, we made this change in ELN stack in the wrong order -> the ELN image now contains `dnf5` and not `dnf4`, so it stays "ready".  Going to update..